e2e/protractor.conf.js

```diff
@@ -24,5 +33,23 @@ exports.config = {
       project: require('path').join(__dirname, './tsconfig.e2e.json')
     });
     jasmine.getEnv().addReporter(new SpecReporter({ spec: { displayStacktrace: true } }));
+    removePreviousDownloads();
   }
 };
+
+function removePreviousDownloads() {
+  try {
+    const downloadsPath = 'e2e/src/downloads';
+    const files = fs.readdirSync(downloadsPath);
```

Comment:
Is this an array of relative file paths, or an array of actual File objects? If it's the first, could you rename to something like relativeFilePaths?

Reply:
it returns an array with the files names rather than paths.